conf/st2.conf.sample

```diff
@@ -301,10 +301,10 @@ facility = local7
 [system]
 # Enable debug mode.
 debug = False
-# True to validate parameters for non-system trigger types when creatinga rule. By default, only parameters for system triggers are validated
-validate_trigger_parameters = False
+# True to validate parameters for non-system trigger types when creatinga rule. By default, only parameters for system triggers are validated.
+validate_trigger_parameters = True
 # True to validate payload for non-system trigger types when dispatching a trigger inside the sensor. By default, only payload for system triggers is validated.
-validate_trigger_payload = False
+validate_trigger_payload = True
```

Comment:
There's not much description on making the validation default. Is it correct to assume this is because we want to validate the parameters and payloads when a trigger is injected manually?

Reply:
Correct - parameters when creating a trigger (so when creating a rule) and payload when a trigger is dispatched (either via webhook or sensor).

We've already done this in the past, but just for system triggers.

Documentation upgrade notes change is available at https://github.com/StackStorm/st2docs/pull/768.